test/test_runner.go

```diff
@@ -478,9 +498,9 @@ func (r *TestRunner) parseAndCheckImport(location common.Location, startCtx runt
 }
 
 // PrettyPrintResults is a utility function to pretty print the test results.
-func PrettyPrintResults(results Results) string {
+func PrettyPrintResults(results Results, scriptPath string) string {
 	var sb strings.Builder
-	sb.WriteString("Test results:\n")
+	sb.WriteString("Test results: " + "(" + scriptPath + ")\n")
```

Comment:
```suggestion
	fmt.Fprintf(&sb, "Test results: %q\n", scriptPath)
```

Reply:
Thanks :pray: Fixed in https://github.com/onflow/cadence-tools/pull/78/commits/b0f0774a5b3365c40d206b226f73beedf69b21e5